aweadmin.py

```diff
@@ -211,6 +258,8 @@ def main():
       showjobs=True
     elif arg == '-n':
```

Comment:
We should really convert these arguments to use the Python argparse module.  But that's outside the scope of this task.